webapp/src/custom_styles.scss

```diff
@@ -0,0 +1,45 @@
+// Custom Bootstrap variable overrides go first
+
+// Colors
+$primary: #223859;
```

Comment:
Please also take this color as theme color in `public/index.html`. Maybe add a comment here to make clear is has to be changed in two places.